proto/pairing/relay.proto

```diff
@@ -9,23 +9,42 @@ service Relayer {
     rpc RelaySubscribe (RelayRequest) returns (stream RelayReply) {}
 }
 
-message RelayRequest {
-    string chainID = 1;
-    string connection_type = 2;
-    string api_url = 3; // some relays have associated urls that are filled with params ('/block/{height}')
-    uint64 session_id = 4;
-    uint64 cu_sum = 5; // total compute unit used including this relay
+message RelaySession {
+    string specID = 1;
+    bytes content_hash = 2;
+    uint64 session_id = 3;
+    uint64 cu_sum = 4; // total compute unit used including this relay
+    string provider = 5;
+    uint64 relay_num = 6;
+    QualityOfServiceReport QoSReport = 7;
+    int64 epoch = 8;
+    bytes unresponsive_providers = 9;

```

Comment:
we are marshalling it because it is sent over grpc

Reply:
Marshaling and unmarshaling into bytes can have some benefits in terms of efficiency and storage size. When using bytes, the data is stored in a compact binary format, which can be more efficient in terms of storage size than storing the data as strings. Additionally, using bytes can also simplify the marshaling and unmarshaling process since the protobuf library can handle the conversion automatically.